mmv1/products/alloydb/Instance.yaml

```diff
@@ -279,6 +279,33 @@ properties:
             values:
               - :ENCRYPTED_ONLY
               - :ALLOW_UNENCRYPTED_AND_ENCRYPTED
+  - !ruby/object:Api::Type::NestedObject
+    name: 'pscInstanceConfig'
+    default_from_api: true
```

Comment:
Is this necessary? I don't see this field getting returned by the API in our nightly builds... default_from_api causes field removal to behave in unexpected ways so we try to avoid it where possible. For more information see https://googlecloudplatform.github.io/magic-modules/develop/field-reference/#default_from_api

Reply:
Removed